op-node/rollup/derive/system_config.go

```diff
@@ -27,6 +28,19 @@ var (
 	ConfigUpdateEventVersion0 = common.Hash{}
 )
 
+var (
+	// A left-padded uint256 equal to 32.
+	OneWordUint = common.Hash{31: 32}
+	// A left-padded uint256 equal to 64.
+	TwoWordUint = common.Hash{31: 64}
+	// 24 zero bytes (the padding for a uint64 in a 32 byte word)
+	Uint64Padding = make([]byte, 24)
+	// 12 zero bytes (the padding for an Ethereum address in a 32 byte word)
+	AddressPadding = make([]byte, 12)
```

Comment:
Can consider making these lowercase / package-level to not make them part of the derivation package API.